### PR TITLE
adding media control feature

### PR DESCRIPTION
**Pull request by qinben10**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "cookie-parser": "^1.4.5",
     "express": "^4.17.1",
     "express-sslify": "^1.2.0",
+    "jquery": "^3.5.1",
```
Comment (adam-adair) [inline package.json]: We shouldn't need to add jquery to this project just to get video/audio controls. It looks like the only use for it is some DOM element class manipulation below. Everywhere else we've done that with vanilla JS

File: server/public/index.html [hunk 2]
```diff
@@ -2,6 +2,8 @@
   <head>
     <link rel="stylesheet" href="style.css">
     <script src='./bundle.js' defer></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://kit.fontawesome.com/648216d052.js" crossorigin="anonymous"></script>
```
Comment (adam-adair) [inline server/public/index.html:5]: I don't know how heavy a font kit is, but also seems like we don't need a full one just for controls. Just add a couple of images to the proejct?

**Review by adam-adair — CHANGES_REQUESTED** [review]
See comments. At this point in the project, we should stick to using vanilla JS instead of jquery

File: package.json (hunk 1 above)
Comment (qinben10) [reply package.json]: done

File: server/public/index.html (hunk 2 above)
Comment (qinben10) [reply server/public/index.html:5]: keep font kit

**Review by adam-adair — APPROVED** [review]
looks good

**Pull request merged**